rectools/model_selection/cross_validate.py

```diff
@@ -49,6 +60,7 @@ def cross_validate(  # pylint: disable=too-many-locals
     k: int,
     filter_viewed: bool,
     items_to_recommend: tp.Optional[ExternalIds] = None,
+    keep_features_for_hot_only: bool = False,
```

Comment:
It still blows my mind :)
Can we name it more meaningful somehow?

`allow_warm_test_inference: bool = True`
`allow_warm_inference: bool = True`
or even
`prefer_warm_inference_over_cold: bool = True`

Reply:
renamed to prefer_warm_inference_over_cold